### ISSUE TITLE
Error relabeling directory

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [projectatomic/libpod#990]
/kind bug

**Description**
While running the baseline test on the latest Podman kit, I received an "Error relabeling directory" error.  I was not able to reproduce it after several attempts and it's the first Podman test done in the baseline tests, all other tests completed successfully.  It smells like a container storage lock error.

**Steps to reproduce the issue:**

1.  Install Fedora 28 on a new VM, `dnf -y update`

2.  ctrid=$(podman pull registry.access.redhat.com/rhscl/redis-32-rhel7)

**Describe the results you received:**
```
########
# Run ls in redis container, this should work
########
ctrid=$(podman pull registry.access.redhat.com/rhscl/redis-32-rhel7)
Trying to pull registry.access.redhat.com/rhscl/redis-32-rhel7...Getting image source signatures
Copying blob sha256:e0f71f706c2a1ff9efee4025e27d4dfd4f328190f31d16e11ef3283bc16d6842
 71.45 MB / 71.45 MB [=====================================================] 13s
Copying blob sha256:121ab4741000471a7e2ddc687dedb440060bf9845ca415a45e99e361706f1098
 1.22 KB / 1.22 KB [========================================================] 0s
Copying blob sha256:033565e6892e5cc6dd03187d00a4575720a928db111274e0fbf31b410a093c10
 7.31 MB / 7.31 MB [========================================================] 1s
Copying blob sha256:f557125ee27db8c621b7865114d0a8f3765c9ab19b08f56d3329c96b3174697e
 6.58 MB / 6.58 MB [========================================================] 1s
Copying config sha256:018297a8d410b6f31eb4996d31a13fc963f43b82c5d387c8bbb7ed81e1fa80ad
 8.80 KB / 8.80 KB [========================================================] 0s
Writing manifest to image destination
Storing signatures
podman run $ctrid ls /
error mounting image volumes: error relabeling directory "/var/lib/containers/storage/overlay-containers/56f849b7b573b34c814b1a737784781fb16b8157d7103661ac78ef274b396667/userdata/volumes/var/lib/redis/data" for volume "/var/lib/redis/data" in container %!q(func() string=0xe768b0): no such file or directory

```

**Describe the results you expected:**

No erro

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
# podman version
Version:       0.6.4
Go Version:    go1.10.3
OS/Arch:       linux/amd64

```

**Output of `podman info`:**

```
podman version
Version:       0.6.4
Go Version:    go1.10.3
OS/Arch:       linux/amd64
[root@localhost ~]# podman info
host:
  MemFree: 960696320
  MemTotal: 2089484288
  SwapFree: 2139865088
  SwapTotal: 2147479552
  arch: amd64
  cpus: 1
  hostname: localhost.localdomain
  kernel: 4.16.15-300.fc28.x86_64
  os: linux
  uptime: 20m 8.01s
insecure registries:
  registries: []
registries:
  registries:
  - docker.io
  - registry.fedoraproject.org
  - quay.io
  - registry.access.redhat.com
  - registry.centos.org
store:
  ContainerStore:
    number: 0
  GraphDriverName: overlay
  GraphOptions:
  - overlay.override_kernel_check=true
  GraphRoot: /var/lib/containers/storage
  GraphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
  ImageStore:
    number: 1
  RunRoot: /var/run/containers/storage

```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Small VM with Fedora 28 installed and updated.

**Comment by TomSweeneyRedHat** [comment]
FWIW, on a newly installed Fedora 27 machine with the same kit but the F27 variant, I ran into the same error in the baseline test.

**Comment by rhatdan** [comment]
@umohnani8 Isn't this an known issue?

**Comment by umohnani8** [comment]
@rhatdan it was, I was seeing it a while back. The way I fixed it was nuking the storage.
https://github.com/projectatomic/libpod/issues/622 - I guess what I was seeing is slightly different that what @TomSweeneyRedHat  is seeing.
But the relabelling issue could be related to selinux? Not sure.

**Comment by rhatdan** [comment]
Yes I believe this is an SELinux issue.  But the error states that the volume directory does not exist.  I thought you were working on supplying builtin images for podman/buildah.

**Comment by umohnani8** [comment]
@rhatdan I don't think this is a built-in volume issue. It is working fine for me on my F28 machine, haven't tried with F27. Smells more like a storage issue as @TomSweeneyRedHat pointed out.
Looking into it, @TomSweeneyRedHat how are you getting version 0.6.4, I am still seeing 0.6.3 even after `dnf update`.

**Comment by rhatdan** [comment]
dnf -y update --enablerepo=updates-testing podman

**Comment by umohnani8** [comment]
Fixed in #1026
